sagemaker-clarify/online_explainability/natural_language_processing/requirements.txt

```diff
@@ -0,0 +1,7 @@
+sagemaker
```

Comment:
should we specify the versions?

Reply:
we actually do a --upgrade, for sagemaker version should not be provided as our notebooks should always work.